CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 
 [comment]: <> (When bumping [pc:VERSION_LATEST_RELEASE] create a new entry below)
 ### Unreleased version
+### v0.8.0
+- Add support for control plane operations for serverless indexes
+- Add support for collections for pod indexes
```

Comment:
I think it's important that we add specifics about the primary class changes here for control plane, what do you think?

- Rename of `PineconeIndexOperationClient` -> `PineconeControlPlaneClient`.
- The change in constructor signatures for this class: `PineconeClientConfig` is no longer required, you now pass `apiKey` as a string directly. However, `PineconeClientConfig` is still needed when connecting to the data plane.

Calling these things out will be important as updating to this version will break users and we want to give them some heads up beyond looking at the integration test implementations.

Reply:
The release notes will have all of the details